examples/with-grafbase/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "next/core-web-vitals"
```

Comment:
Should be removed as per https://github.com/vercel/next.js/blob/canary/contributing/examples/adding-examples.md